.github/workflows/tests.yml

```
@@ -65,7 +85,7 @@ jobs:
     if: github.event_name == 'pull_request' && needs.tap_syntax.outputs.run-tests
```

This is a total guess, I don't know if this will work:

```suggestion
    if: >
      github.event_name == 'pull_request' && needs.tap_syntax.outputs.run-tests && !(
        matrix.version == '11-arm' && needs.tap_syntax.outputs.skip-arm
      )
```

If not, another option might be to add the `!(matrix.version == '11-arm' && needs.tap_syntax.outputs.skip-arm)` to each step. This isn't ideal because it will result in the workflow showing up as "succeeded" when in fact it was "skipped".